tests/test_encoder.py

```diff
@@ -213,3 +213,20 @@ def test_encode_property_list(self):
             expected_geometry=[[0, 0]],
             properties=properties,
             expected_properties={'test_empty': ''})
+
+    def test_encode_multiple_values(self):
+        geometry = 'POINT(0 0)'
+        properties = dict(foo='bar')
```

Comment:
what about a round trip encode-decode, you could then easily handle the order no?